### PR TITLE
feat: add review put and get

### PR DESCRIPTION
**Pull request by spaceragga**

### CODE REVIEW

File: src/resources/authentication/checkAuthentication.js [hunk 1]
```diff
@@ -29,6 +30,14 @@ const checkAuthentication = (req, res, next) => {
     return next();
   }
 
+  if (req.path === REVIEW_PATH && req.method === 'GET') {
+    return next();
+  }
+
+  if (req.path === REVIEW_PATH && req.method === 'PUT') {
```
Comment (aleksei-bulgak) [inline src/resources/authentication/checkAuthentication.js:37]: only authorized users should be able to post comments. with this condition you allow to leave comments for everyone

File: src/resources/review/review.db.repository.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const Review = require('./review.model');
+
+const get = async () => {
+  const review = await Review.find();
+
+  return review;
+};
+
+const put = async arrReview => {
```
Comment (aleksei-bulgak) [inline src/resources/review/review.db.repository.js:9]: as I understand `arr` prefix stands for array. In your case this is a single entity so you can use singular form

File: src/resources/review/review.model.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const ReviewSchema = new Schema(
+  {
+    review: {
```
Comment (aleksei-bulgak) [inline src/resources/review/review.model.js:6]: text message should be required as we do not want to store empty comments
same for name. user without name or anonymous comments should not be allowed

File: src/resources/authentication/checkAuthentication.js (hunk 1 above)
Comment (spaceragga) [reply src/resources/authentication/checkAuthentication.js:37]: button "submit new review" available only autorized user

**Pull request merged**